### PR TITLE
Add support for 0.5-beta

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
Using manifest data from https://github.com/lightningnetwork/lnd/releases/download/v0.5-beta/manifest-v0.5-beta.txt.

### CODE REVIEW

**Comment by OttoAllmendinger** [comment]
I used the following script to update `manifest.json`, maybe you can integrate it

```
const fs = require('fs');
const { promisify } = require('util');
const axios = require('axios');
const https = require('https');

const fetchManifest = async (version) => {
  const manifestUrl =
    `https://github.com/lightningnetwork/lnd/releases/download/` +
    `${version}/manifest-${version}.txt`;
  const res = await axios.get(manifestUrl);
  return res.data.split('\n')
    .filter((line) => !line.match(/^\s*$/))
    .map((line) => {
      const [hash, filename] = line.split(/\s\s*/);
      if (!hash || !filename || hash.length !== 64) {
        throw new Error('invalid line: ' + line);
      }
      return [hash, filename];
    })
    .reduce((obj, [hash, filename]) => Object.assign(obj, {[hash]: filename }), {});
};

const updateManifest = async (version) => {
  const localManifestPath = 'config/manifest.json';
  const localManifest = JSON.parse(await promisify(fs.readFile)(localManifestPath));
  const newManifest = {
    [version]: await fetchManifest(version)
  };
  Object.keys(localManifest).forEach((oldVersion) => {
    newManifest[oldVersion] = localManifest[oldVersion];
  });
  await promisify(fs.writeFile)(localManifestPath, JSON.stringify(newManifest, null, 2));
};

updateManifest('v0.5-beta');
```

**Comment by mrfelton** [comment]
Thanks @OttoAllmendinger 

Need to update the test cases though.

The expectation is that by default, it installs the latest version. The test cases are expecting that to be 0.4.2-beta still.

eg:
https://github.com/mrfelton/lnd-binary/blob/master/test/install.js#L17
https://github.com/mrfelton/lnd-binary/blob/master/test/install.js#L47

etc

**Comment by OttoAllmendinger** [comment]
derp - didn't think of running the tests

will fix

File: config/manifest.json [hunk 1]
```diff
@@ -1,4 +1,26 @@
 {
+  "v0.5-beta": {
```
Comment (OttoAllmendinger) [inline config/manifest.json]: the leading `v` is wrong here

**Comment by mrfelton** [comment]
Nice one. Can you squash the commits please @OttoAllmendinger - then I think this is good to merge.

**Pull request merged**